setup.py

```diff
@@ -21,11 +21,12 @@
     'Topic :: Software Development :: Libraries :: Application Frameworks',
 ]
 
+# ToDo: django>=1.8.17,<1.9 is <1.9 really a hard requirement?
```

Comment:
@DanielPollithy can you elaborate on the django requirement? is 1.8 the only supported version?